netplan_cli/cli/ovs.py

```diff
@@ -43,6 +42,19 @@ class OvsDbServerNotRunning(Exception):
     pass
 
 
+def _assert_ovsdb_server_connection():
+    """Invoke OPENVSWITCH_OVS_VSCTL, raise OvsDbServerNotRunning on error."""
+    try:
+        # The `get-manager` command is a light weight database operation which
+        # we use as indication of the client being able to connect.
+        subprocess.check_call(
+            [OPENVSWITCH_OVS_VSCTL, '--timeout', '5', 'get-manager'])
+    except (subprocess.CalledProcessError, OSError):
+        raise OvsDbServerNotRunning('{} is unable to connect to database, '
```

Comment:
note: This might have some implications for https://github.com/canonical/netplan/pull/421